djangosaml2/tests/__init__.py

```diff
@@ -462,6 +462,9 @@ def test_assertion_consumer_service(self):
         user_id = self.client.session[SESSION_KEY]
         user = User.objects.get(id=user_id)
         self.assertEqual(user.username, "student")
+        # Since a new user object is created, the password
+        # field is set to have an unusable password.
+        self.assertEqual(user.has_usable_password(), False)
```

Comment:
@peppelinux I piggy backed on an existing test case to verify that new user will have an unusable password. No change would be made for user objects where password is already set.